src/widget/woverview.cpp

```diff
@@ -396,6 +409,30 @@ void WOverview::onPassthroughChange(double v) {
     update();
 }
 
+void WOverview::onEndOfTrackBlinkTimerChange(bool v) {
```

Comment:
```suggestion
void WOverview::onEndOfTrackBlinkTimerChange(double v) {
```

Reply:
correct. I wonder why the compiler or CI didn't complain about the signal parameter mismatch. Implicit conversion?